src/sonic-yang-models/yang-models/sonic-extension.yang

```diff
@@ -7,10 +7,18 @@ module sonic-extension {
 
     description "Extension yang Module for SONiC OS";
 
+    revision 2021-02-23 {
+        description "Adding extensions as per Sonic Yang Guidelines";
+    }
+
     revision 2019-07-01 {
         description "First Revision";
     }
 
+	/* For complete guide of using these extensions in SONiC yangs, refer
+	SONiC yang guidelines at
+	https://github.com/Azure/SONiC/blob/master/doc/mgmt/SONiC_YANG_Model_Guidelines.md */
+
     extension key-regex-configdb-to-yang {
         description "Key regex used to convert config DB keys to YANG Config";
         argument "value";
```

Comment:
I believe, we dont need these extensions any more, please remove if not required.

Reply:
Praveen already raised PR to remove these extensions (https://github.com/Azure/sonic-buildimage/pull/6915).

Reply:
ok